astroquery/sdss/core.py

```diff
@@ -30,6 +30,184 @@
+               'segue2_target2', 'boss_target1', 'eboss_target0',
+               'ancillary_target1', 'ancillary_target2', 'primtarget',
+               'sectarget', 'spectrographid', 'sourcetype', 'targettype',
+               'ra', 'dec', 'cx', 'cy', 'cz', 'xfocal', 'yfocal',
+               'lambdaeff', 'bluefiber', 'zoffset', 'z', 'zerr', 'zwarning',
+               'class', 'subclass', 'rchi2', 'dof', 'rchi2diff', 'z_noqso',
+               'zerr_noqso', 'zwarning_noqso', 'class_noqso',
+               'subclass_noqso', 'rchi2diff_noqso', 'z_person',
+               'class_person', 'comments_person', 'tfile', 'tcolumn_0',
+               'tcolumn_1', 'tcolumn_2', 'tcolumn_3', 'tcolumn_4',
+               'tcolumn_5', 'tcolumn_6', 'tcolumn_7', 'tcolumn_8',
+               'tcolumn_9', 'npoly', 'theta_0', 'theta_1', 'theta_2',
+               'theta_3', 'theta_4', 'theta_5', 'theta_6', 'theta_7',
+               'theta_8', 'theta_9', 'veldisp', 'veldisperr', 'veldispz',
+               'veldispzerr', 'veldispchi2', 'veldispnpix', 'veldispdof',
+               'wavemin', 'wavemax', 'wcoverage', 'snmedian_u',
+               'snmedian_g', 'snmedian_r', 'snmedian_i', 'snmedian_z',
+               'snmedian', 'chi68p', 'fracnsigma_1', 'fracnsigma_2',
+               'fracnsigma_3', 'fracnsigma_4', 'fracnsigma_5',
+               'fracnsigma_6', 'fracnsigma_7', 'fracnsigma_8',
+               'fracnsigma_9', 'fracnsigma_10', 'fracnsighi_1',
+               'fracnsighi_2', 'fracnsighi_3', 'fracnsighi_4',
+               'fracnsighi_5', 'fracnsighi_6', 'fracnsighi_7',
+               'fracnsighi_8', 'fracnsighi_9', 'fracnsighi_10',
+               'fracnsiglo_1', 'fracnsiglo_2', 'fracnsiglo_3',
+               'fracnsiglo_4', 'fracnsiglo_5', 'fracnsiglo_6',
+               'fracnsiglo_7', 'fracnsiglo_8', 'fracnsiglo_9',
+               'fracnsiglo_10', 'spectroflux_u', 'spectroflux_g',
+               'spectroflux_r', 'spectroflux_i', 'spectroflux_z',
+               'spectrosynflux_u', 'spectrosynflux_g', 'spectrosynflux_r',
+               'spectrosynflux_i', 'spectrosynflux_z', 'spectrofluxivar_u',
+               'spectrofluxivar_g', 'spectrofluxivar_r',
+               'spectrofluxivar_i', 'spectrofluxivar_z',
+               'spectrosynfluxivar_u', 'spectrosynfluxivar_g',
+               'spectrosynfluxivar_r', 'spectrosynfluxivar_i',
+               'spectrosynfluxivar_z', 'spectroskyflux_u',
+               'spectroskyflux_g', 'spectroskyflux_r', 'spectroskyflux_i',
+               'spectroskyflux_z', 'anyandmask', 'anyormask', 'platesn2',
+               'deredsn2', 'snturnoff', 'sn1_g', 'sn1_r', 'sn1_i', 'sn2_g',
+               'sn2_r', 'sn2_i', 'elodiefilename', 'elodieobject',
+               'elodiesptype', 'elodiebv', 'elodieteff', 'elodielogg',
+               'elodiefeh', 'elodiez', 'elodiezerr', 'elodiezmodelerr',
+               'elodierchi2', 'elodiedof', 'htmid', 'loadversion', 'img']
+
```

Comment:
Can these be retrieved from the website?  If so, it would be good to have a separate script to retrieve them if they are likely to change.  We could store in a JSON file and read it in here.  I would still want to include the data in the repository even if we do have a retrieval script, though.

This is not a requirement, just a suggestion